packages/story-editor/src/app/api/apiProvider.js

```diff
@@ -35,20 +34,14 @@ function APIProvider({ children }) {
     withoutImages: [],
   });
 
-  actions.getPageTemplates = useCallback(
-    async ({ showImages = false } = {}) => {
-      // check if pageTemplates have been loaded yet
-      if (pageTemplates.current.base.length === 0) {
-        pageTemplates.current.base = await getAllTemplates({ cdnURL });
-        pageTemplates.current.withoutImages = removeImagesFromPageTemplates(
-          pageTemplates.current.base
-        );
-      }
-
-      return pageTemplates.current[showImages ? 'base' : 'withoutImages'];
-    },
-    [cdnURL]
-  );
+  actions.getPageTemplates = useCallback(async () => {
+    // check if pageTemplates have been loaded yet
+    if (pageTemplates.current.base.length === 0) {
+      pageTemplates.current.base = await getAllTemplates({ cdnURL });
+    }
+
+    return pageTemplates.current.base;
```

Comment:
```suggestion
    // check if pageTemplates have been loaded yet
    if (pageTemplates.current.length === 0) {
      pageTemplates.current= await getAllTemplates({ cdnURL });
    }

    return pageTemplates.current;
```

we could now simplify this ref a bit (see also comment above)